OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs

```diff
@@ -22,8 +22,12 @@ public async Task ShouldThrowDependencyExceptionOnAddWhenSqlExceptionOccursAndLo
             AssignmentAttachment randomAssignmentAttachment = CreateRandomAssignmentAttachment();
             AssignmentAttachment inputAssignmentAttachment = randomAssignmentAttachment;
             var sqlException = GetSqlException();
```

Comment:
The left-hand side has to be specific. Feel free to scope it in a separate PR too.